Multisig/Data/Services/Notification Service/UnregisterNotificationTokenRequest.swift

```diff
@@ -17,9 +17,9 @@ struct UnregisterNotificationTokenRequest: JSONRequest {
 
     typealias ResponseType = EmptyResponse
 
-    init(deviceID: UUID, address: Address) {
+    init(deviceID: String, address: Address) {
         self.address = address.checksummed
-        self.deviceID = deviceID.uuidString.lowercased()
+        self.deviceID = deviceID
```

Comment:
the `lowercased()` got lost - either when this init is called or in here it should be present

Reply:
I checked - it worked fine. But I will make it lowercased()